ui/test/nuxeo-date.test.js

```diff
@@ -0,0 +1,87 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { html, fixture } from '@nuxeo/testing-helpers';
+import moment from '@nuxeo/moment';
+import '../widgets/nuxeo-date.js';
+
+function setNuxeoConfigDateFormat(format, value) {
+  window.Nuxeo = window.Nuxeo || {};
+  window.Nuxeo.UI = window.Nuxeo.UI || {};
+  window.Nuxeo.UI.config = window.Nuxeo.UI.config || {};
+  window.Nuxeo.UI.config[format] = window.Nuxeo.UI.config[format] || value;
+}
+
+suite('nuxeo-date', async () => {
+  test('Should hide the nuxeo-tooltip if provided format and tooltipFormat are equal', async () => {
+    const element = await fixture(
+      html`
+        <nuxeo-date datetime=${moment()} format="LLL" tooltipFormat="LLL"></nuxeo-date>
+      `,
+    );
+
+    const tooltip = element.shadowRoot.querySelector('nuxeo-tooltip');
+    expect(tooltip.hidden).to.be.true;
```

Comment:
Maybe here we could use the helper `isElementVisible` to make sure it is not visible in the DOM. This question follows a series of discussions we had as a team where we questioning if testing the property as we are doing here is not like testing the bindings themselves (which in that case wouldn't be our responsibility to test).
```suggestion
    expect(isElementVisible(tooltip)).to.be.false;
```

Reply:
It is a good point for further thinking. 🙂 
Still, looking to `nuxeo-tooltip` code, it may be a little bit trickier, since it has `:host { display: none; }`. 
This means that it will be always "not visible".

Reply:
Also, it may seem that we are testing the binding, but we `nuxeo-date` are the ones defining it. So, I guess I'm ok with that, @semisse .
The alternative would be to repeat the `nuxeo-tooltip.test.js` code here, right? (cc @andreacornaglia , @semisse )

Reply:
Exactly, what the nuxeo-date function is doing is adding the property hidden to the tooltip, which is what we're testing. Element visibility will depend on the tooltip functionality and we're already testing that. So I would keep the test as is.